antiplag/views.py

```diff
@@ -12,18 +12,28 @@
 from django.conf import settings
 from nlp.text_preprocessing import preprocess_text
 from django.utils.translation import ugettext as _
+from django.core.exceptions import ValidationError
+from django.core.validators import validate_email
+import json, requests
 
 
 class SubmissionList(APIView):
     serializer_class = serializers.SubmissionSerializer
-    parser_classes = (MultiPartParser, FormParser)
+    parser_classes = (MultiPartParser, FormParser, JSONParser)
```

Comment:
Is the JSONParser used anywhere? Would it work without it?

Reply:
removed